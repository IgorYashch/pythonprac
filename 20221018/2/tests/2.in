:sin x sin(x)
:sqrt z sqrt(z)
:exp l exp(l)
exp 2
sin 17
sqrt 4
exp 12
quit           "hello {}{} hello"