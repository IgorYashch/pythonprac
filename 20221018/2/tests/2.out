7.38905609893065
-0.9613974918795568
2.0
162754.79141900392
hello 48 hello
